### ISSUE TITLE
feature(v8) Make keyboard navigation working correctly

### ISSUE THREAD

**Issue by gpbl** [gpbl/react-day-picker#1031]
In v8, users could navigate the calendar with arrow keys, while in v8, the navigation between days works using the <kbd>TAB</kbd> key.

This feature from v7 should be replicated in v8 too!

### Notes

- It should be possible to copy/paste the implementation from v7
- https://github.com/gpbl/react-day-picker/issues/1030 should be implemented first

### To-do

- [ ] Close #1030
- [ ] Implement new keyboard navigation
- [ ] Add tests

**Comment by gpbl** [comment]
See some inputs from https://github.com/gpbl/react-day-picker/issues/783#issuecomment-574851613
